data/quickstarts/gpu-enabled-notebook-quickstart.yaml

```diff
@@ -0,0 +1,42 @@
+kind: ConsoleQuickStart
+metadata:
+  name: gpu-enabled-notebook-quickstart
+  annotations:
+    opendatahub.io/categories: 'Getting started,GPUs,Python'
+spec:
+  displayName: Creating a Machine Learning Model using the NVIDIA GPU Add-on.
+  appName: jupyterhub
```

Comment:
should it be `nvidia`?

Reply:
They use caps on their site.  I think leaving caps is correct.

Reply:
changed the appName to be `nvidia`